frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -15,57 +18,157 @@ type MessageDetailsProps = {
+        expanded: false,
+        content: items,
+        headingLevel: "h3",
+        className: "",
+    });
+
+    return results;
+};
+
 export function MessageDetails() {
     const navigate = useNavigate();
     const { id } = useParams<MessageDetailsProps>();
     const { messageDetails } = useMessageDetails(id!!);
+    const warnings: WarningError[] = messageDetails
+        ? messageDetails.warnings
+        : [];
+    const errors: WarningError[] = messageDetails ? messageDetails.errors : [];
+
+    const fileUrl = messageDetails?.fileUrl ? messageDetails.fileUrl : "";
+    const { fileLocation, sendingOrg, fileName } = parseFileLocation(fileUrl);
 
     return (
         <>
             <div
                 className="grid-container margin-bottom-10"
                 data-testid="container"
             >
-                <div className="margin-bottom-5">
-                    <h1>Message Details</h1>
-                    <Button onClick={() => navigate(-1)} type="button">
+                <div>
+                    <h2>Message ID Search</h2>
+                    <Button
+                        onClick={() => navigate("/admin/message-tracker")}
+                        type="button"
+                    >
                         Back
                     </Button>
                 </div>
-                <DetailItem
-                    item={"Message ID"}
-                    content={messageDetails!.messageId}
-                ></DetailItem>
+                <hr className="margin-top-4 margin-bottom-4" />
+                <div className="display-flex flex-column">
+                    <span className="text-base font-sans-sm">Message ID</span>
+                    <h1 className="margin-top-1">
+                        {messageDetails!.messageId}
+                    </h1>
+                </div>
+                <hr />
                 <div className="display-flex flex-column margin-bottom-5">
-                    <h3>Sender:</h3>
-                    <DetailItem
-                        item={"Sender Name"}
-                        content={messageDetails!.sender}
-                    ></DetailItem>
-                    <DetailItem
-                        item={"Incoming Report ID"}
-                        content={messageDetails!.reportId}
-                    ></DetailItem>
-                    <DetailItem
-                        item={"Incoming File Name"}
-                        content={messageDetails!.fileName}
-                    ></DetailItem>
-                    <DetailItem
-                        item={"Incoming File URL"}
-                        content={messageDetails!.fileUrl}
-                    ></DetailItem>
+                    <h2 className="font-sans-lg margin-bottom-4">Sender:</h2>
+                    <div className="font-sans-sm bg-gray-5 radius-lg padding-top-3 padding-bottom-4 padding-left-3 padding-right-3">
+                        <DetailItem
+                            item={"Submitter"}
```

Comment:
This also existed beforehand, but we don't need to wrap string props in curly braces:

```jsx
item="Submitted"
```